js-apps/meep-frontend/src/js/containers/idc-map.js

```diff
@@ -193,7 +344,11 @@ class IDCMap extends Component {
     });
 
     if (existingMarker === undefined) {
-      // Create new UE marker & path
+      // Create path, if any
+      // var p = ue.path ? L.GeoJSON.geometryToLayer(ue.path) : null;
+      var p = pathLatLngs ? L.polyline(pathLatLngs) : null;
+
```

Comment:
Shouldn't we set the UE asset name in the path options?

Reply:
Added as suggested.